src/contracts/libraries/SlashingAccountingUtils.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+/**
+ * @title Library of utilities for calculations related to slashing in EigenLayer
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+library SlashingAccountingUtils {
+
+    // TODO: set this based on payments epochs
+    uint256 internal constant EPOCH_GENESIS_TIMESTAMP = 1606824000;
```

Comment:
perfer another library for epoch logic

Reply:
Yes and have PaymentCoordinator read GENESIS TIMESTAMP from the same library

Reply:
epoch logic now has its own library https://github.com/Layr-Labs/eigenlayer-contracts/pull/543/commits/5074afb01bdc7ae00a47866b2699424b5b780234